integrations/ultralytics/train.py

```diff
@@ -50,6 +50,7 @@
 from utils.plots import plot_images, plot_labels, plot_results, plot_evolution
 from utils.torch_utils import ModelEMA, select_device, intersect_dicts, torch_distributed_zero_first
 
+from sparseml.pytorch.nn import replace_activations
```

Comment:
Quick thought, would it make sense to put the imports under our commented blocks/where the code is used? Ideally, that way it makes it easier to swap out -- I'm not sold on needing to do that, though. Just a thought

Reply:
good idea. done.